operator/konflux.bundle.Dockerfile

```diff
@@ -14,6 +14,21 @@ RUN make bundle-post-process
 
 FROM scratch
 
+# Enterprise Contract labels.
+LABEL com.redhat.component="rhacs-operator-bundle-container"
+LABEL description="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL distribution-scope="public"
+LABEL io.k8s.description="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL name="rhacs-operator-bundle"
+# Release label is required by EC although has no practical semantics.
+# We also set it to not inherit one from a base stage in case it's RHEL or UBI.
+LABEL release="1"
+LABEL url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c"
+LABEL vendor="Red Hat, Inc."
+LABEL version="${MAIN_IMAGE_TAG}"
+LABEL maintainer="Red Hat, Inc."
+LABEL summary="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+
```

Comment:
Please add `LABEL`s: 
- `io.openshift.tags`
- `io.k8s.display-name`
- `com.redhat.license_terms`
- `source-location`

so that the operator bundle is similar to our other Konflux containers.

Reply:
Done